frontend/src/scenes/cohorts/Cohorts.tsx

```diff
@@ -142,7 +137,7 @@ export function Cohorts(): JSX.Element {
                                 <ExportOutlined />
                             </Tooltip>
                         </a>
-                        {cohort.id !== 'new' && cohort.id !== 'personsModalNew' && (
+                        {cohort.id !== 'new' && (
```

Comment:
Can we perform the same isNew check here as we do below? `typeof cohort.id === 'number'`